src/Agent/NewRelic/Agent/Core/NewRelic.Agent.Core/AgentManager.cs

```diff
@@ -281,6 +281,7 @@ private void LogInitialized()
                     }
                 }
 
+                Log.Debug($".NET Runtime Version (from System.Environment.Version): {System.Environment.Version}");
```

Comment:
Not a requirement, but just a question... Should we only do this for the NETSTANDARD build of the agent?

Reply:
We already have some code in DotnetVersion.cs that looks up .net versions, we might just able to add the logging code there, or you can look up the data already stored in AgentInstallConfiguration (although you might need to store the raw return value of that environment property in AgentInstallConfiguration).

Reply:
That code only gets you the generic version for Framework, not the specific build version. It could get the major.minor.patch version for .NET Core, but so can either of the two methods I've proposed. 

I'm testing out some changes where I added `System.Runtime.InteropService.RuntimeInformation` to the ILRepack list, seems to work just fine so far.

Reply:
If that works, should we refactor the code in DotnetVersion to leverage that too (with a separate PR later)?

Reply:
I think we could do that. The ILRepack works fine, check my latest commit.